holdings_maintenance/pds4/pds4linkshelf.py

```
@@ -0,0 +1,1270 @@
+                            (delta / 86400.), force=True)
+            else:
+                logger.info('!!! Link shelf file is out of date %.1f minutes' %
+                            (delta / 60.), force=True)
+
+            dt = datetime.datetime.now()
+            os.utime(link_path)
+            os.utime(link_pypath)
+            logger.info('!!! Time tag on link shelf files set to',
+                        dt.strftime('%Y-%m-%dT%H-%M-%S'), force=True)
+        else:
+            logger.info(f'!!! Link shelf file is up to date; repair canceled',
```

Flake8 (7.3.0)

[error] f-string is missing placeholders

(F541)